app/controllers/amr_heatmap_controller.rb

```diff
@@ -54,6 +54,7 @@ def amr_counts
     render json: amr_data
   end
 
-  def amr_heatmap
+  def visualization
```

Comment:
I would move this controller function to the top and call it `index`.

Reply:
maybe this is more of a 'show' action? since an 'index' would be for seeing many AMR Heatmaps?